Mathlib/Geometry/Euclidean/Angle/Oriented/Affine.lean

```diff
@@ -212,6 +212,19 @@ theorem oangle_sign_eq_zero_iff_collinear {p₁ p₂ p₃ : P} :
     (∡ p₁ p₂ p₃).sign = 0 ↔ Collinear ℝ ({p₁, p₂, p₃} : Set P) := by
   rw [Real.Angle.sign_eq_zero_iff, oangle_eq_zero_or_eq_pi_iff_collinear]
 
+/-- An oriented angle is not zero and `π` if and only if the three points are not collinear. -/
+theorem oangle_ne_zero_and_ne_pi_iff_not_collinear {p₁ p₂ p₃ : P} :
+  ∡ p₁ p₂ p₃ ≠ 0 ∧ ∡ p₁ p₂ p₃ ≠ π ↔ ¬ Collinear ℝ {p₁, p₂, p₃} := by
+  rw [oangle_ne_zero_and_ne_pi_iff_affineIndependent, affineIndependent_iff_not_collinear_set]
+
+/-- If two oriented angles are equal, one triple is affinely independent if and only if
+the other is. -/
+theorem affineIndependent_iff_of_oangle_eq {p₁ p₂ p₃ p₁' p₂' p₃' : P}
+  (h_eq : ∡ p₁ p₂ p₃ = ∡ p₁' p₂' p₃') :
```

Comment:
Since we have the stronger lemma `affineIndependent_iff_of_two_zsmul_oangle_eq`, I'm doubtful of adding this variant as well. (If at the site of use you do have `h` saying the two angles are equal, not just equal mod pi, then you should be able to pass `h ▸ rfl` to that stronger lemma.)